scripts/dump_data.js

```diff
@@ -0,0 +1,167 @@
+const _ = require("lodash");
+const AdmZip = require("adm-zip");
+const fs = require("fs/promises");
+const inquirer = require("inquirer");
+const path = require("path");
+const { list: listFiles } = require("recursive-readdir-async");
+
+const knex = require("../src/server/db/connection");
+const { uploadFSName } = require("../src/server/services/persist-upload");
+const { periodTemplatePath } = require("../src/server/services/get-template");
+const { UPLOAD_DIR } = require("../src/server/environment");
+
+const TABLES = [
+  "users",
+  "agencies",
+  "reporting_periods",
+  "uploads", // has FK to reporting_periods
+  "arpa_subrecipients", // has FK to uploads
+  "application_settings", // has FK to reporting_periods
+  "projects", // has FK to reporting_periods
+  "period_summaries", // has FK to reporting_periods, projects
+];
+
+async function getDatabaseContents() {
+  console.log("Fetching all DB rows...");
+  const start = Date.now();
+
+  const data = await Promise.all(
+    TABLES.map((tableName) => knex(tableName).select("*"))
+  );
+  const ret = _.fromPairs(_.zip(TABLES, data));
```

Comment:
It might be a useful/simple sanity check to print out the number of rows in each table here, just so we can quickly compare it to a similar printout at the import side